File: 110algoritmos/exercicios007/ex007-Csharp/ex007.cs
using System;

class Program {

    static void Main() {
        float n1, n2, media;

        Console.WriteLine("Digite a primeira nota: ");
        n1 = int.Parse(Console.ReadLine());
        
        Console.Write("Digite a segunda nota: ");
        n2 = int.Parse(Console.ReadLine());

        media = (n1 + n2)/2;

        Console.WriteLine("A média entre " + n1 + " e " + n2 + " eh igual a " + media);

    }
}

File: 110algoritmos/exercicios007/ex007-Go/ex007.go
package main

import "fmt"

func main() {
	var n1 float32
	var n2 float32

	fmt.Println("Digite a primeira nota: ")
	fmt.Scan(&n1)

	fmt.Println("Digite a segunda nota: ")
	fmt.Scan(&n2)

	media := (n1 + n2) / 2

	fmt.Printf("A média entre %v e %v é %.1f", n1, n2, media)

}
